controllers/storageoscluster_controller.go

```diff
@@ -66,8 +67,14 @@ func (r *StorageOSClusterReconciler) SetupWithManager(mgr ctrl.Manager, kubeVers
 	_, span, _, log := instrumentation.Start(context.Background(), "StorageosCluster.SetupWithManager")
 	defer span.End()
 
+	// Set channel 1.18 if k8s version is not greater or equal to v1.19.0
+	channel := "stable"
+	if stable := version.IsSupported(kubeVersion, "v1.19.0"); !stable {
```

Comment:
This should go into main, but could you please leaf at least a TODO to flip the check? If version have a custom directory use it otherwise stable. I think the existing solution would only work until we have a new custom channel. WDYT?

Reply:
Agreed, this is a short-term solution. Will leave a  TODO